### PR TITLE
Collapsable layout for DE and EN FAQ

### PR DESCRIPTION
**Pull request by mairima**
To Do: 
English translation missing
Add bootstrap container

### CODE REVIEW

File: app/views/pages/faq.en.html.erb [hunk 1]
```diff
@@ -1,102 +1,198 @@
...
+        <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+          <h4>Who can register on Speakerinnen.org?</h4>
+        </button>
+      </h2>
     </div>
-    <div class="col-lg-6 col-md-12">
-      <%= image_tag('faq_login_en.png', class:'border img-fluid') %>
+    <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordionExample">
+      <div class="card-body">
+        <p>YThe goal of Speakerinnen.org is to get more women on stages and podiums, to conferences, workshops, meetings, and into media. Hence, everybody who identifies as a women can register herself. The Speakerinnen team vets every submission for completeness before publishing it.</p>
```
Comment (sabrina-ulbrich) [inline app/views/pages/faq.en.html.erb]: typo at the beginning: YThe

File: app/views/pages/faq.en.html.erb [hunk 2]
```diff
@@ -1,102 +1,198 @@
...
+    <div class="card-header" id="headingFour">
+      <h2 class="mb-0">
+        <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+          <h4>Why is my profile not yet published?</h4>
+        </button>
+      </h2>
+    </div>
+    <div id="collapseFour" class="collapse" aria-labelledby="headingFour" data-parent="#accordionExample">
+      <div class="card-body">
+       <p>WThe Speakerinnen team consists of volunteers and is not constantly online. If you just submitted your profile, it can take a moment before it is reviewed and published. If you only provided very little information, we may assume that it's not yet complete. If you want to publish it anyways, send a mail to <a href=”mailto:team@speakerinnen.org”>team@speakerinnen.org</a>.</p>
```
Comment (sabrina-ulbrich) [inline app/views/pages/faq.en.html.erb]: typo at the beginning: WThe

Comment (zaziemo) [reply app/views/pages/faq.en.html.erb]: good catch :)

**Comment by zaziemo** [comment]
@mairima finally we finished the work you started :) It is now online!

**Pull request merged**